CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - offer to open http-links detected as proxy also in the browser (#6237)
 - Show notifications for reactions on own messages (#2331)
+- Dark mode and adaptive app icon (#2425)
```

Comment:
After reading this Changelog-entry I expected DeltaChat to have a dark mode — and not just icon for dark and tinted. So maybe something like this?

```suggestion
- Icons for Dark-mode and tint colors (#2425)
```